$NetBSD: patch-ac,v 1.3 2010/06/20 22:20:18 tron Exp $

Avoid linking "tree-inline.o" twice which breaks the build under Mac OS X.

--- gcc/cp/Make-lang.in.orig	2008-12-05 08:01:58.000000000 +0000
+++ gcc/cp/Make-lang.in	2010-06-20 23:11:07.000000000 +0100
@@ -72,8 +72,8 @@
 # Shared with C front end:
 CXX_C_OBJS = attribs.o c-common.o c-format.o c-pragma.o c-semantics.o c-lex.o \
 	c-dump.o $(CXX_TARGET_OBJS) c-pretty-print.o c-opts.o c-pch.o \
-	incpath.o cppdefault.o c-ppoutput.o c-cppbuiltin.o prefix.o \
-	c-gimplify.o c-omp.o tree-inline.o
+	incpath.o c-ppoutput.o c-cppbuiltin.o prefix.o \
+	c-gimplify.o c-omp.o
 
 # Language-specific object files for C++ and Objective C++.
 CXX_AND_OBJCXX_OBJS = cp/call.o cp/decl.o cp/expr.o cp/pt.o cp/typeck2.o \
